### ISSUE TITLE
Update inbox notification documentation

### ISSUE THREAD

**Issue by becdetat** [woocommerce/woocommerce-admin#7185]
The documentation here (https://woocommerce.github.io/woocommerce-admin/#/examples/activity-panel-inbox) is out of date, per a conversation here (p9h1Lv-2qu-p2 comment hash `#comment-7177` @Dan-Q @bgrgicak). Please update it to the latest features and method names.

**Comment by jyggiz** [comment]
Hey, can I take this issue?

**Comment by becdetat** [comment]
@jyggiz definitely! I've just updated the description a bit, I think it would be good to include one or more links to actual live examples of note creation. 

Thanks for getting involved :-D
